src/App/Containers/Wip/wip.jsx

```diff
@@ -7,8 +7,9 @@ import Button       from 'App/Components/Form/button.jsx';
 import Icon         from 'Assets/icon.jsx';
 import { connect }  from 'Stores/connect';
 
-const onClick = () => {
-    window.location.href = window.open(urlFor('trading', undefined, undefined, true));
+const onClick = (e) => {
+    e.preventDefault();
+    window.open(urlFor('trading', undefined, undefined, true), '_blank');
```

Comment:
We set the title as `_blank` while redirecting, it will be overridden once the other page loads right?

Reply:
I tested it, should be fine.

Reply:
`_blank` is the target